csrc/ir/nodes.cpp

```diff
@@ -2361,9 +2361,46 @@ LoadStoreOp::LoadStoreOp(
   addDataAttribute(cache_op);
 }
 
+namespace {
+// Returns the permutation from `in` to `out`, i.e., `out[perm[i]]==in[i]`.  As
+// a precondition, `out` must be a permutation of `in` per the definition of
+// std::is_permutation.
+template <typename T>
+std::vector<int64_t> Permutation(
```

Comment:
nit: I think function names should start with lower case letters?

Reply:
Done.